tabpfn_client/prompt_agent.py

```diff
@@ -51,7 +51,7 @@ def prompt_and_set_token(cls, user_auth_handler: "UserAuthenticationClient"):
         # Registration
         if choice == "1":
             # validation_link = input(cls.indent("Please enter your secret code: "))
-            validation_link = "tabpfn-2023"
+            validation_link = "tabpfn-test"
```

Comment:
The production is still using `tabpfn-2023`, right?

Reply:
In the server, should we just keep the `validation_link` the same for all `DEV`, `PREPROD`, and `PROD`?
Or actually, do we still need this `validation_link` feature?

Reply:
let's keep it synced. i will update it to be 2023 in DEV too